### ISSUE TITLE
用語集 注記4の「失敗例」は「失敗」とすべきでは

### ISSUE THREAD

**Issue by momdo** [waic/wcag22#301]
https://waic.jp/translations/WCAG22/#h-note-94

> テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、<mark>失敗例</mark>となる。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も<mark>失敗例</mark>となる。

https://www.w3.org/TR/2023/REC-WCAG22-20231005/#h-note-94

> It is a <mark>failure</mark> if no background color is specified when the text color is specified, because the user's default background color is unknown and cannot be evaluated for sufficient contrast. For the same reason, it is a <mark>failure</mark> if no text color is specified when a background color is specified.
